--- /vendor/api-platform/core/src/GraphQl/Action/EntrypointAction.php	2020-08-13 08:22:33.000000000 +0200
+++ /vendor/api-platform/core/src/GraphQl/Action/EntrypointAction.php	2020-08-13 08:21:56.000000000 +0200
@@ -15,7 +15,7 @@

 use ApiPlatform\Core\GraphQl\ExecutorInterface;
 use ApiPlatform\Core\GraphQl\Type\SchemaBuilderInterface;
-use GraphQL\Error\Debug;
+use GraphQL\Error\DebugFlag as Debug;
 use GraphQL\Error\Error;
 use GraphQL\Error\UserError;
 use GraphQL\Executor\ExecutionResult;
